docs/source/format/Metadata.rst

```diff
@@ -192,13 +189,74 @@ categories:
 
 Refer to `Schema.fbs`_ for up-to-date descriptions of each built-in
 logical type.
+Custom Application Metadata
+---------------------------
+
+We provide a ``custom_metadata`` field at three levels to provide a
+mechanism for developers to pass application-specific metadata in
+Arrow protocol messages. This includes ``Field``, ``Schema``, and
+``Message``.
+
+The colon symbol ``:`` is to be used as a namespace separator. It can
+be used multiple times in a key.
+
+The ``ARROW`` pattern is a reserved namespace for internal Arrow use
+in the ``custom_metadata`` fields. For example,
+``ARROW:extension:name``.
+
+Extension Types
+---------------
+
+User-defined "extension" types can be defined setting certain
+``KeyValue`` pairs in ``custom_metadata` in the ``Field`` metadata
```

Comment:
"\`" is missing after "custom_metadata\`".

Reply:
fixed